vault.js

```diff
@@ -1,4 +1,17 @@
 'use strict';
 module.exports = function() {
-  
+    var fart={};
+    function setValue(key, value){
+        fart[key] = value;
+    }
+    function getValue(key){
```

Comment:
Your indentation is set to 4 spaces for some reason, we need to see if your editor is setup correctly.